vmm/src/config.rs

```diff
@@ -2102,6 +2114,18 @@ impl VmConfig {
     pub fn is_tdx_enabled(&self) -> bool {
         self.platform.as_ref().map(|p| p.tdx).unwrap_or(false)
     }
+
+    // Release resource hold by VmConfig.
+    pub fn release(&mut self) {
+        let mut net_devices = self.net.clone();
```

Comment:
I'm afraid this leaves a set of stale FDs in its source location. That's not good.

Reply:
Thanks, fixed.